src/app/views/provisioning/subscriptions.less

```diff
@@ -1,5 +1,10 @@
 #provisioning-subscriptions {
+ width: 90vw;
```

Comment:
`width: 90vw;` after a certain point will be too large Please add a `max-width` corresponding to the widest bootstrap breakpoint.

Reply:
Hey @alexnoox ,

I have tested in different screens, different zooms, and different browser sizes and seems to work fine. 
90vw will set the width to the 90% of the viewport, so this will be the minimum and maximum relative to the viewport. Am I wrong?

I have changed it anyway, but it would be great if you can get back to me on this.